packages/tools/pie-icons-webc/build.js

```diff
@@ -0,0 +1,113 @@
+import pieIcons from '@justeattakeaway/pie-icons';
+import { pascalCase } from 'pascal-case';
+import path from 'path';
+import fs from 'fs-extra';
+import kebabCase from 'kebab-case';
+
+const { icons } = pieIcons.default;
+
+const componentTemplate = (name, svg) => {
+    const [, svgClasses] = svg.match(/class="(.+?)"/);
+
+    return `import { getSvgProps } from '@justeattakeaway/pie-icons-configs';
+
+const template = document.createElement('template');
+template.innerHTML = '${svg}';
+
+export class ${name} extends HTMLElement {
+    constructor () {
+        super();
+        const clone = template.content.cloneNode(true);
+        this.root = this.attachShadow({ mode: 'open' });
+        this.root.append(clone);
+    }
+
+    static get observedAttributes () {
+        return ['size'];
+    }
+
+    get size () {
+        return this.getAttribute('size');
+    }
+
+    set size (value) {
+        this.setAttribute('size', value);
+    }
+
+    get class () {
+        return this.getAttribute('class');
+    }
+
+    set class (value) {
+        this.setAttribute('class', value);
+    }
+
+    connectedCallback () {
+        const svg = this.root.querySelector('svg');
+        const svgSize = getSvgProps('${svgClasses}', '', null, '${name}');
+        svg.setAttribute('width', svgSize.width);
+        svg.setAttribute('height', svgSize.height);
+        this.setAttribute('class', '${svgClasses}');
+        this.root.append(svg);
+    }
+
+    attributeChangedCallback (attr, oldVal, newVal) {
+        const svg = this.root.querySelector('svg');
+        let svgSize;
+
+        if (attr === 'size') {
+            svgSize = getSvgProps('${svgClasses}', '', newVal, '${name}');
```

Comment:
I see that second parameter here passed as an empty string meaning that no `staticClasses` will be applied to the component. Maybe in web components classes are appended by default and it is not needed but worth checking.